.gitignore

```diff
@@ -12,6 +12,7 @@
 data/*.txt
 data/*.ttf
 data/*.sfd
+/docs/
```

Comment:
It goes with this line in `Doxyfile`:
```
+OUTPUT_DIRECTORY       = docs
```
It contains `docs/html`, `docs/man` etc.